test/test_plugin_systemd_jruby.rb

```diff
@@ -16,8 +16,6 @@ def setup
     skip_unless_signal_exist? :TERM
     skip_unless :jruby
 
-    super
-
```

Comment:
Could this fix go into master? I noticed this test failing in master (https://github.com/puma/puma/actions/runs/7160776392/job/19495517337#step:8:874) but I couldn't make sense of it (systemd plugin should not be loaded on JRuby, code has guard against `JRUBY_VERSION` constant)

Reply:
I guess not, I see more test have the same `super` call, it is how `TestIntegration` works. https://github.com/puma/puma/blob/08e71a2c6aaccf295bea909aacf40a9aa86b3702/test/helpers/integration.rb#L27 (Sorry)

Still curious if you @MSP-Greg have any theories about the `TestPluginSystemdJruby` master failures :)